FlutterFire.md

```diff
@@ -0,0 +1,34 @@
+# FlutterFire
```

Comment:
I'd probably also add a note about how official these plugins are. Are these officially from the Firebase team? We can see what they use for wording for their PolymerFire, etc projects.